### ISSUE TITLE
add "hot spring"

### ISSUE THREAD

**Issue by akthom** [ESIPFed/sweet#135]
For new term requests, please provide the following information:

## Preferred term label 

Hot spring

## Synonyms

Geothermal spring

## Textual definition

A hot spring is a spring produced by the emergence of geothermally heated groundwater that rises from the Earth's crust. While some of these springs contain water that is a safe temperature for bathing, others are so hot that immersion can result in injury or death. (https://en.wikipedia.org/wiki/Hot_spring)

## Suggested parent term

Please look in the hierarchy in a browser such as [OLS](http://www.ebi.ac.uk/ols/ontologies/sweet)
Terrestrial hydrosphere?

## Attribution

0000-0001-6238-3498

**Comment by nicholascar** [comment]
Using the annotation suggestions from https://github.com/ESIPFed/sweet/wiki/SWEET-Class-Annotations-Proposal, I think this proposal, annotated at the whole-of-class level, as opposed to that wiki page's example for just a definition, is as follows. Note that I suggest the term [Spring](http://sweetontology.net/realmHydroBody/Spring) be used as the parent:

```

###  http://sweetontology.net/realmHydroBody/HotSpring
soreahb:HotSpring
    a owl:Class ;
    rdfs:subClassOf soreahb:Spring
    skos:prefLabel "hot spring"@en ;
    skos:altLabel "geothermal spring"@en ;
    skos:definition  "A hot spring is a spring produced by the emergence of geothermally heated groundwater that rises from the Earth's crust. While some of these springs contain water that is a safe temperature for bathing, others are so hot that immersion can result in injury or death."@en ;
    dct:source <https://en.wikipedia.org/wiki/Hot_spring> ;
    dct:created "2019-07-19"^^xsd:date ;
    dct:creator <http://orcid.org/0000-0001-6238-3498> ;
.
```

**Comment by brandonnodnarb** [comment]
Addressed with #180.
